config/sync/core.extension.yml

```diff
@@ -7,7 +7,6 @@ module:
   automated_cron: 0
   big_pipe: 0
   block: 0
-  block_api: 0
```

Comment:
Your PR should be clean.
I should not see any unrelated changes to the ticket.

Reply:
Okay done.

Reply:
Now I do not see the entry for form_api module.

Reply:
Okay, have done that now.